### PR TITLE
Bugfix set template.order to 1 by default.

### PR DESCRIPTION
**Pull request by simitt**
To ensure default order is not changed to 0, set it to 1 in default config.

### CODE REVIEW

**Comment by simitt** [comment]
jenkins, test this

File: libbeat/template/template_test.go [hunk 1]
```diff
@@ -81,12 +82,13 @@ func TestTemplate(t *testing.T) {
...
 	ver := common.MustNewVersion("6.6.0")
-	template, err := New(beatVersion, beatName, *ver, TemplateConfig{}, false)
-	assert.NoError(t, err)
+	template, err := New(beatVersion, beatName, *ver, DefaultConfig(), false)
+	require.NoError(t, err)
 
 	data := template.Generate(common.MapStr{}, nil)
 	assert.Equal(t, []string{"testbeat-6.6.0-*"}, data["index_patterns"])
 	meta, err := data.GetValue("mappings.doc._meta")
 	assert.NoError(t, err)
```
Comment (urso) [inline libbeat/template/template_test.go]: please replace assert with require as well.

**Comment by urso** [comment]
Do we need a changelog here?

**Comment by simitt** [comment]
jenkins, test this.

**Pull request merged**